README.md

```
@@ -1,173 +1,99 @@
 
-Meridian stands on excellent open-source work:
+See [CONTRIBUTING.md](CONTRIBUTING.md) for guidelines, maintainers, and how to submit PRs.
 
-- [**screenpipe**](https://screenpi.pe) — the capture crates Meridian's in-process capture is forked from (pinned at the last MIT release, 0.4.6).
-- [**Tauri**](https://tauri.app) — the framework that wraps the dashboard and tray into a single native app.
-- [**candle**](https://github.com/huggingface/candle) — the Rust ML framework running Meridian's local text-embedding model.
+Thanks to all contributors:
 
-Thank you to these communities. 🙏
+<a href="https://github.com/Meridiona/meridian/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=Meridiona/meridian" />
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)